.circleci/src/workflows/scheduled_delivery.yml

```diff
@@ -249,7 +249,7 @@ jobs:
       context:
         - mattermost
         - deliverino
-      production_delivery: true
+      production_delivery: false
```

Comment:
This flag is now always false, right? Should we remove that functionality?

Reply:
Not for triggered deliveries. I think we should still keep this as we may need it sometime.